Workbooks/Azure Monitor - Workspaces/Agent Health/Agent Health.workbook

```diff
@@ -0,0 +1,615 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Agent health\r\nUse this report to learn about the health of the agents in your workspace. \r\n<br/>\r\n<br/>"
+      },
+      "name": "text - 0"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "parameters": [
+          {
+            "id": "cb5e40b4-020b-4f01-88a6-50e6225e2fb9",
+            "version": "KqlParameterItem/1.0",
+            "name": "DefaultSubscription_Internal",
+            "type": 1,
+            "isRequired": true,
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| take 1\r\n| project subscriptionId",
+            "crossComponentResources": [
+              "value::selected"
+            ],
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "20fb47ba-f521-47ea-9fb7-66339ad9c56a",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "type": 6,
+            "isRequired": true,
+            "query": "summarize by subscriptionId\r\n| project value = strcat(\"/subscriptions/\", subscriptionId), label = subscriptionId, selected = iff(subscriptionId =~ '{DefaultSubscription_Internal}', true, false)\r\n",
```

Comment:
Shall we only list Subscriptions that have Workspaces? Actually wonder if a Subscription picker really needed... Just showing all workspaces may be ok, most customers have a reasonable # of these, at least in default subscriptions.

Reply:
Maybe. But for now I am simply following what we are doing in all other templates at Azure Monitor level.